stream_alert/rule_processor/config.py

```diff
@@ -35,22 +35,23 @@ def load_config(conf_dir='conf/'):
     key denotes the name of the log type, and includes 'keys' used to match
     rules to log fields.
     """
-    conf_files = {
-        'sources': 'sources.json',
-        'logs': 'logs.json'
-    }
-    config = {}
-    for desc, filename in conf_files.iteritems():
-        with open(os.path.join(conf_dir, filename)) as data:
+    conf_files = {'sources', 'logs'}
```

Comment:
this can be a tuple - this allows us to preserve the order and it's not going to be used with other `set` methods